src/server/set_family.cc

```diff
@@ -633,7 +629,7 @@ OpResult<uint32_t> OpAdd(const OpArgs& op_args, std::string_view key, ArgSlice v
     res = AddStrSet(op_args.db_cntx, vals, UINT32_MAX, &co);
   }
 
-  db_slice.PostUpdate(op_args.db_cntx.db_index, it, key, !new_key);
+  add_res.post_updater.Run();
```

Comment:
I dont think there is a reason to run it explicitly here. We can run it at the destructor